lib/dom.dart

```diff
@@ -433,7 +428,7 @@ class Text extends Node {
 
   String get data => _data = _data.toString();
   set data(String value) {
-    _data = value ?? '';
+    _data = identical(value, null) ? '' : value;
```

Comment:
Probably worth a comment here and a TODO to remove this eventually?

Reply:
Added a comment.